tcms/testruns/tests/test_permissions.py

```diff
@@ -56,3 +58,39 @@ def verify_post_with_permission(self):
         self.assertRedirects(
             response,
             reverse('testruns-get', args=[self.test_run.pk]))
+
+
+class TestRunTestCase(tests.PermissionsTestCase):
```

Comment:
-> `CreateTestRunViewTestCase` (convention is: name of view under test + "TestCase")